packages/sdk-js/src/GrowthBook.ts

```diff
@@ -241,12 +313,21 @@ export class GrowthBook<
 
   public subscribe(cb: SubscriptionFunction): () => void {
     this._subscriptions.add(cb);
-
     return () => {
       this._subscriptions.delete(cb);
     };
   }
 
+  private _canSubscribe() {
+    return this._ctx.backgroundSync !== false && this._ctx.subscribeToChanges;
+  }
+
+  private async _refreshForRemoteEval() {
+    if (!this._ctx.remoteEval) return;
+    if (!this._loadFeaturesCalled) return;
+    await this._refresh({}, false, true);
```

Comment:
Will this promise ever reject?  If so, we should catch it here

Reply:
It will throw if missing clientKey or if trying to use remoteEval on our cloud servers. I think it would be best to allow it to error rather than catching. Wdty

Reply:
The problem is we're not awaiting this function when we call it from `setUrl`, etc..  So if it throws, devs might not notice unless they happen to look in their browser console.

I would think for these 2 use cases you listed, that we could catch that during the constructor, which will be a lot more visible during development.

Reply:
ok, suppressed the errors and moved them to the constructor for remoteEval